storage.js

```diff
@@ -63,7 +63,7 @@ module.exports = function (generate) {
   }
 
   exports.load = function(filename, cb) {
-    filename = toFile(filename, 'secret')
+    filename = toFile(filename)
```

Comment:
:fire:  Haven't reviewed what this change does / if is safe

Reply:
Removed because `toFile()` only accepts one argument: https://github.com/ssbc/ssb-keys/blob/22a9efbca3d0ad1d0cbf194a3fa68c3c107abfc0/storage.js#L17